client/components/Slide/Components/AudioResponse/index.js

```diff
@@ -1,8 +1,9 @@
 import { type } from './type';
 export { type };
-export const defaultValue = () => ({
-    type,
-    prompt: 'Provide a prompt for users to respond to here.'
+export const defaultValue = ({ responseId }) => ({
+    prompt: 'Provide a prompt for users to respond to here.',
+    responseId,
```

Comment:
Added `responseId` to all response components, because it's created by the slide editor at the time they are added to a slide.